packages/core/src/ceramic.ts

```diff
@@ -92,19 +102,22 @@ const tryDocId = (id: string): DocID | null => {
  * `$ npm install --save @ceramicnetwork/core`
  */
 class Ceramic implements CeramicApi {
-  private readonly _docmap: Record<string, Document>
   private readonly _doctypeHandlers: Record<string, DoctypeHandler<Doctype>>
 
   public readonly pin: PinApi
   public readonly context: Context
 
+  private readonly _docCache: DocCache
+
   // TODO: Make the constructor private and force the use of Ceramic.create() everywhere
   constructor (public dispatcher: Dispatcher,
                public pinStore: PinStore,
                context: Context,
                private _networkOptions: CeramicNetworkOptions,
-               private _validateDocs: boolean = true) {
-    this._docmap = {}
+               private _validateDocs: boolean = true,
```

Comment:
We're probably getting close to the point where it would make sense to have a CeramicCreateOptions struct that we pass in, rather than an ever growing list of optional positional args.  Doesn't have to happen in this PR though

Reply:
Oh actually, we basically have that already via the `CeramicConfig` passed to `Ceramic.create`.  The real thing to do is probably what the `TODO` above says, make this constructor private and force everything to go through Ceramic.create.  That would let us put all "default" behaviors in `Ceramic.create` and make all the constructor args here non-optional

Reply:
created a [740](https://github.com/ceramicnetwork/js-ceramic/issues/740) story for that